_timeline/2015-01-01-Hyberboria.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Hyperboria
+date: 2015-01-01
+---
+Toronto-based individuals are involved with the international cjdns mesh networking community--[Hyperboria](https://hyperboria.net/).
```

Comment:
I joined Hyperboria in June 2013, but it was just me and ircerr for a long time after that. I guess 2015 is when you connected?